src/jit/utils.cpp

```diff
@@ -1604,6 +1605,10 @@ unsigned __int64 FloatingPointUtils::convertDoubleToUInt64(double d) {
     // is unsigned. gcc converts a negative double to zero when
     // cast to an unsigned. To make gcc conform to MSVC behavior
     // this cast is necessary.
-    u64 = UINT64(INT64(d));     
+    u64 = UINT64(INT64(d));
```

Comment:
Could you update this comment? I would say that the only "spec hole" is that it isn't clear that the "overflow" case would apply to converting a negative double to int.
I would reword this to say something like (also note that it should be "Ecma" not "ECMA"):

While the Ecma spec doesn't specifically call this out, the case of conversion from negative double to unsigned is effectively an overflow and therefore the result is unspecified.
With MSVC, such a conversion results in the bit-equivalent unsigned value of the conversion to integer.
Other compilers convert negative doubles to zero when the target is unsigned.
To make the behavior consistent across OS's on _TARGET_XARCH_, this double cast is needed to make gcc conform to MSVC behavior.